docs/API.md

```diff
@@ -438,6 +438,7 @@ Options can be configured as a parameter to the [init()](./interfaces/tracer.htm
 | protocolVersion | `DD_TRACE_AGENT_PROTOCOL_VERSION` | `0.4`          | Protocol version to use for requests to the agent. The version configured must be supported by the agent version installed or all traces will be dropped. |
 | runtimeMetrics  | `DD_RUNTIME_METRICS_ENABLED`      | `false`        | Whether to enable capturing runtime metrics. Port 8125 (or configured with `dogstatsd.port`) must be opened on the agent for UDP. |
 | profiling       | `DD_PROFILING_ENABLED`            | `false`        | Whether to enable profiling. |
+| ci              | `DD_CI_VISIBILITY_ENABLED`        | `false`        | Whether running in a CI environment. |
```

Comment:
Could this use the `CI` environment variable instead? All CI products I know of are setting this environment variable automatically. For example: https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables. Or is the idea to enable the feature? In that case, the description is incorrect as you could enable this locally. Also, if this is disabled, is it expected that traces are generated for test runners? This would be inconsistent if enabling the feature results in the traces disappearing.

Reply:
This is confusing, my bad. For CI Visibility to work the customer needs to set specific configuration to `mocha`, `cucumber`, `jest` and `cypress`, so I don't think there needs to be an automatic detection based on the environment: the user already needs to do something for CI Visibility to work. We can use the new entrypoint for CI Visibility (`dd-trace/ci/init`) to "tell the tracer" to process spans differently (that is, set `_dd.origin` for every span). 

I'd say it's better *not* to expose this API at all: if the customer is following the instructions to instrument a testing framework, then CI Visibility is enabled.

Reply:
It looks like the option is still exposed.

Reply:
ci visibility, as a user of the tracer, needs a way to start it in `ci` mode. Unless we find another way (https://github.com/DataDog/dd-trace-js/pull/1734#discussion_r781553730 maybe?), I can't think of a way not to expose it